osu.Game/Overlays/BeatmapSet/Scores/NotSupporterPlaceholder.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+
+namespace osu.Game.Overlays.BeatmapSet.Scores
+{
+    public class NotSupporterPlaceholder : Container
+    {
+        public NotSupporterPlaceholder()
+        {
+            LinkFlowContainer text;
+
+            AutoSizeAxes = Axes.Both;
+            Child = new FillFlowContainer
+            {
+                AutoSizeAxes = Axes.Both,
+                Direction = FillDirection.Vertical,
+                Spacing = new Vector2(0, 10),
+                Children = new Drawable[]
+                {
+                    new OsuSpriteText
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Text = @"You need to be an osu!supporter to access the friend and country rankings!",
+                        Font = OsuFont.GetFont(weight: FontWeight.Bold),
+                    },
+                    text = new LinkFlowContainer(t => t.Font = t.Font.With(size: 12))
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Direction = FillDirection.Horizontal,
+                        AutoSizeAxes = Axes.Both,
+                    }
+                }
+            };
+
+            text.AddText("Click ");
+            text.AddLink("here", "/home/support");
```

Comment:
Why's this not a full URL like here?
https://github.com/ppy/osu/blob/fd3e2375bb166624783f8f3676e70493868ab565/osu.Game/Screens/Menu/Disclaimer.cs#L123

Reply:
Inspired by
https://github.com/ppy/osu/blob/8da65fc9094282a458662d32c4aeb55deccfa721/osu.Game/Overlays/AccountCreation/ScreenWarning.cs#L27